### ISSUE TITLE
Añadir tutoriales

### ISSUE THREAD

**Issue by laplaces-entropy** [laplaces-entropy/R-TMOLoader#8]
El programa además de ser poco intuitivo, no tiene ningún lugar con tutoriales o documentación.
**Intentaré hacer videos.**

**Comment by Khalel-Albus** [comment]
Recomendacion, podrias hacer un documento de texto que explique lo que le vayas agregando una especie de changelog con tutorial todo en uno que se pueda descargar desde la misma herramienta

**Comment by laplaces-entropy** [comment]
> Recomendacion, podrias hacer un documento de texto que explique lo que le vayas agregando una especie de changelog con tutorial todo en uno que se pueda descargar desde la misma herramienta

Claro, no veo porque no... Lo intentare después de los errores críticos que mencionaste ( #9 ) y de los tutoriales que digo aqui

**Comment by Khalel-Albus** [comment]
claro, es mas facil que hacer videos, es comodo que la gente pueda acceder al tuto sin necesidad de desplazarse mucho la comodidad siempre es la premisa para los usuarios, solo la parte del tutorial no la pongas demasiado tecnica para que sea facil de entender, suerte! espero se pueda arreglar tambien lo del issue #9

**Comment by laplaces-entropy** [comment]
> claro, es mas facil que hacer videos, es comodo que la gente pueda acceder al tuto sin necesidad de desplazarse mucho la comodidad siempre es la premisa para los usuarios, solo la parte del tutorial no la pongas demasiado tecnica para que sea facil de entender, suerte! espero se pueda arreglar tambien lo del issue #9

Lo que aún estoy pensando es lo de la voz, no quiero usar la mía por qué no se me entiende bien, pero con texto dos un poco complicado de entender... Estoy buscando un sintetizador de voz que se escuche natural.

**Comment by Khalel-Albus** [comment]
yo decia mas preparar un documento txt con el contenido del changelog y el tutorial explicativo que la gente pueda leer tranquilamente, porque si pones un video en el repositorio o un audio no significa eso que cada que se abra la app lo descargara? al menos un documento no es tanto no pesa nada

**Comment by laplaces-entropy** [comment]
> yo decia mas preparar un documento txt con el contenido del changelog y el tutorial explicativo que la gente pueda leer tranquilamente, porque si pones un video en el repositorio o un audio no significa eso que cada que se abra la app lo descargara? al menos un documento no es tanto no pesa nada

No planeo descargar el vídeo, tal vez algún link o pomelo en el README de este repositorio

**Comment by Khalel-Albus** [comment]
ahh eso si podria ser, los principiantes veran el video solo si creen que lo necesitan y ya, la verdad la app es bastante intuitiva, lo uni que podria confunder porque en un principio me confundio es lo de la cantidad de procesos, pero de resto es muy entendible todo
